warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_stale_service_alerts.sql

```diff
@@ -0,0 +1,71 @@
+{{ config(
+    materialized='incremental',
+    incremental_strategy='insert_overwrite',
+    partition_by = {
+        'field': 'date',
+        'data_type': 'date',
+        'granularity': 'day',
+    },
+) }}
+
+{% if is_incremental() %}
+    {% set timestamps = dbt_utils.get_column_values(table=this, column='date', order_by = 'date DESC', max_records = 1) %}
+    {% set max_ts = timestamps[0] %}
+{% endif %}
+
+WITH
+
+feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index_sa') }}
+    {% if is_incremental() %}
+    WHERE date >= EXTRACT(DATE FROM TIMESTAMP('{{ max_ts }}'))
+    {% else %}
+    WHERE date >= DATE_SUB(CURRENT_DATE(), INTERVAL {{ var('RT_LOOKBACK_DAYS') }} DAY)
+    {% endif %}
+),
+
+stg_gtfs_rt__service_alerts AS (
+    SELECT * FROM {{ ref('stg_gtfs_rt__service_alerts') }}
+    {% if is_incremental() %}
+    WHERE dt >= EXTRACT(DATE FROM TIMESTAMP('{{ max_ts }}'))
+    {% else %}
+    WHERE dt >= DATE_SUB(CURRENT_DATE(), INTERVAL {{ var('RT_LOOKBACK_DAYS') }} DAY)
+    {% endif %}
+),
+
+service_alert_ages AS (
+    SELECT
+        dt,
+        base64_url,
+        COUNT(*) AS num_service_alerts,
+        -- I'm not sure what timestamp to use for service alerts - I don't even understand yet how this check applies to service alerts
```

Comment:
I think that for service alerts we should just use header timestamp. `active.start` and `active.end` can cover, like, weeks (ex. for construction). Also `active.start` and `active.end` are arrays.